lib/IRGen/GenClangDecl.cpp

```diff
@@ -119,6 +119,10 @@ class ClangDeclFinder
     return true;
   }
 
+  bool TraverseRequiresExpr(clang::RequiresExpr *RE) {
+    return false;
```

Comment:
Actually this should be `true`. if you return `false` the traversal exits early which will make us miss some symbols. Can you test that in your test? e.g. make your function call something else and make sure that's emitted

Reply:
Right, thanks for noticing this!